packages/.vitepress/theme/components/FunctionsList.vue

```diff
@@ -15,12 +14,56 @@ useEventListener('click', (e) => {
     window.dispatchEvent(new Event('hashchange'))
 })
 
-const query = useUrlSearchParams('hash-params', { removeFalsyValues: true })
-const search = toRef(query, 'search') as Ref<string | null>
-const category = toRef(query, 'category') as Ref<string | null>
-const hasComponent = toRef(query, 'component') as any as Ref<boolean>
-const hasDirective = toRef(query, 'directive') as any as Ref<boolean>
-const sortMethod = toRef(query, 'sort') as Ref<'category' | 'name' | 'updated' | null>
+const isMounted = useMounted()
+
+const query = useUrlSearchParams<{
+  search: string | null
+  category: string | null
+  component: boolean
+  directive: boolean
+  sort: 'category' | 'name' | 'updated' | null
+}>('hash-params', { removeFalsyValues: true })
+
+const search = computed<string | null>({
+  get() {
+    return isMounted.value ? query.search : null
+  },
```

Comment:
I believe this syntax looks much clearer in this case
```suggestion
  get: () =>  isMounted.value ? query.search : null,
```

Reply:
Thank you for your suggestion. I've edited it because it's definitely easier to understand.
https://github.com/vueuse/vueuse/pull/4428/commits/a2fd49fd49097db83f7aa663d37054609eaf27f0